### PR TITLE
solution exo9

### PR DESCRIPTION
**Pull request by tianyuchen**

### CODE REVIEW

File: 09-Palindrome_Number.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Explanation: From left to right, it reads -121. From right to left, it becomes
+121-. Therefore it is not a palindrome.
+Example 3:
+
+Input: 10
+Output: false
+Explanation: Reads 01 from right to left. Therefore it is not a palindrome.
+Follow up:
+
+Coud you solve it without converting the integer to a string?
```
Comment (gameboy1024) [inline 09-Palindrome_Number.py:23]: For questions that are this simple, you should look at follow up questions.
**Coud you solve it without converting the integer to a string?**

**Review by gameboy1024 — CHANGES_REQUESTED** [review]

File: 09-Palindrome_Number.py (hunk 1 above)
Comment (tianyuchen) [reply 09-Palindrome_Number.py:23]: done :)

File: 09-Palindrome_Number.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        return str(x) == str(x)[::-1]
+
+# Solution without converting the integer to a string
+class Solution2:
+    def isPalindrome(self, x):
+        """
+        :type x: int
+        :rtype: bool
+        """
+        reversedNum = 0
```
Comment (gameboy1024) [inline 09-Palindrome_Number.py]: There is a code style guidance to declare a variable close to where you use it.
In this case, you should move the declarations before your loop.
There's no need to declare these if x < 0.

**Review by gameboy1024 — APPROVED** [review]

File: 09-Palindrome_Number.py (hunk 2 above)
Comment (tianyuchen) [reply 09-Palindrome_Number.py]: done

**Pull request merged**